src/emulator/pubsubEmulator.ts

```diff
@@ -62,7 +70,19 @@ export class PubsubEmulator implements EmulatorInstance {
   }
 
   async stop(): Promise<void> {
-    await downloadableEmulators.stop(Emulators.PUBSUB);
+    try {
+      await downloadableEmulators.stop(Emulators.PUBSUB);
+    } catch (e: unknown) {
+      this.logger.logLabeled("DEBUG", "pubsub", JSON.stringify(e));
+      exec(PUBSUB_KILL_COMMAND, (err, stdout) => {
```

Comment:
Yeah, here - if we do this on Windows, are issues silently logged and it moves on?

Reply:
Added a windows check and will follow-up if we see any windows issues (and test once I have a machine to play with)